met-web/src/components/engagement/view/widgets/Subscribe/SubscribeWidget.tsx

```diff
@@ -87,8 +87,12 @@ function SubscribeWidget() {
                     and Protection of Privacy Act, for the purpose of providing content updates and future opportunities to participate.\
                     Your email is never shared with third parties.',
                     '',
-                    'If you have any questions about the collection, use and disclosure of your personal information,\
-                    please contact the Director of Digital Services at Sid.Tobias@gov.bc.ca',
+                    <Typography sx={{ fontSize: '0.8rem' }}>
```

Comment:
Since we're allowing to pass JSX here could we consider this pattern:
![image](https://user-images.githubusercontent.com/91914654/233675309-f7380f8b-32e8-4e02-aea8-d9543063a43f.png)

Reply:
learnMoreBox here is just one element that we can handle however.

Reply:
Ok cool let me see where i can refactor this

Reply:
@jadmsaadaot updated it let me know what you think :)